### PR TITLE
Donate Help Page FA Form basic rendering

### PR DESCRIPTION
**Pull request by danielfmiranda**
# Description

Link to sample test page: https://donate-foundation-s-11173-dona-jtqs8w.mofostaging.net/en/help/
Related PRs/issues: #11173

This PR handles the basic rendering of the unstyled FA form.

# Screenshots

Screenshot 2023-10-25 at 20-13-25 Donate Help

### CODE REVIEW

**Review by mmmavis — CHANGES_REQUESTED** [review]
Hey @danielfmiranda , 

At a glance it's not possible to tell what code is copied from FormAssembly vs our own custom code so it's tricky to do code review. Could you mimic what we have for petition form ([formassembly_body.html](https://github.com/MozillaFoundation/foundation.mozilla.org/blob/main/network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/formassembly_body.html#L3-L48) and [formassembly_head.html](https://github.com/MozillaFoundation/foundation.mozilla.org/blob/main/network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/formassembly_head.html#L3-L26)) and add notes to the top of the two FA related files? Having notes/instructions will also help future code edits from breaking the form.

We'll also need to edit default CSP values on various files for recaptcha to work!

Thanks!

File: network-api/networkapi/templates/donate/fragments/formassembly_body.html [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+                    'es-MX': 'tfa_231',
+                    'es-AR': 'tfa_231',
+                    'es-ES': 'tfa_231',
+                    'es-CL': 'tfa_231'
+                }
+
+    // Assign the locale, or default to 'Other' if the locale isn't supported.
+                varLocale.value = localeMap[locale] || 'tfa_227';
+            });
+
```
Comment (danielfmiranda) [inline network-api/networkapi/templates/donate/fragments/formassembly_body.html:73]: Hi @mmmavis just calling out this custom localization JS that I found from Mia and the thunderbird team. I am probably going to have to update this in my localization PR as they are referencing locales we do not use on the foundation site (ex: en-US, es-MX, etc.).

**Comment by danielfmiranda** [comment]
Hi @mmmavis! Thanks for the review. 

I have updated this PR with what we discussed on slack which includes:
- Updated env.default to include the URLs needed to render the form / recaptcha
- Some comments at the top of the files, which will be updated as I make more changes to the form such as styling/localization


This pr is now ready for re-review whenever you find a free moment. Thanks!

**Review by mmmavis — CHANGES_REQUESTED** [review]
Looking great! If we can update CSP configs in `app.json` and `continous-integration.yml` as well, that'd be rad 💯

**Comment by danielfmiranda** [comment]
Thanks for the review @mmmavis! Just updated both `app.json` and `continuous-integration.yml` 👍

**Review by mmmavis — APPROVED** [review]
💯

**Pull request merged**